index.js

```diff
@@ -2,12 +2,13 @@
 var path = require('path');
 var fs = require('graceful-fs');
 var mkdirp = require('mkdirp');
-var objectAssign = require('object-assign');
+var assign = require('object-assign');
```

Comment:
I prefer the names to be the camelCased version of the actual import. This makes it more explicit and easier to read the code.